doc/api/events.md

```diff
@@ -139,10 +140,9 @@ myEmitter.emit('error', new Error('whoops!'));
   // Throws and crashes Node.js
 ```
 
-To guard against crashing the Node.js process, developers can either register
-a listener for the `process.on('uncaughtException')` event or use the
-[`domain`][] module (_Note, however, that the `domain` module has been
-deprecated_).
+To guard against crashing the Node.js process, a listener can be registered
+for the `process.on('uncaughtException')` event or the [`domain`][] module can be
```

Comment:
"the `process.on('uncaughtException')` event" is confusing since it's a piece of code and not just the event name.